releasenotes/notes/logs-error-status-99afc61f269a25df.yaml

```diff
@@ -0,0 +1,4 @@
+enhancements:
+  - |
+    Improve Logs agent status by adding errors when breaking the agent.
```

Comment:
Releasenotes are public (in the Changelog), could we better define what `breaking the agent` means, specify that we're talking abour the log-agent and maybe use a less scary wording (`Improve Logs agent status by displaying errors the Logs agent` for example).

Reply:
Thanks, did not know. Updating it!

Reply:
Updated